docker-compose.yml

```diff
@@ -77,5 +75,80 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  monitoring_otel_collector:
+    container_name: monitoring_otel_collector
+    image: otel/opentelemetry-collector-contrib:0.98.0
+    networks:
+      - altinntestlocal_network
+    command:
+      - --config=/etc/otel-collector.yaml
+    volumes:
+      - ./infra/otel-collector.yaml:/etc/otel-collector.yaml
+    ports:
+      - "1888:1888"   # pprof extension
+      - "8888:8888"   # Prometheus metrics exposed by the collector
+      - "8889:8889"   # Prometheus exporter metrics
+      - "13133:13133" # health_check extension
+      - "4317:4317"   # OTLP gRPC receiver
+      - "55679:55679" # zpages extension
```

Comment:
This is a lot of ports to expose externally on the host machine. Are every one required to be accessible from user browser/locally running app? Especially `8888` seems like something that a dev would use for another purpose.